splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -275,9 +275,10 @@ def _dump_with_migrated_entities(
     _collapse_entities(
         global_config.content["pages"].get("inputs", {}).get("services"), entity_type
     )
-    _collapse_entities(
-        global_config.content["pages"]["configuration"].get("tabs"), entity_type
-    )
+    if global_config.has_configuration():
```

Comment:
`_collapse_entities` does accept `None` as a first option and it immediately returns, we can remove this `if` statement.

Reply:
Made the changes.